### DISCUSSION TITLE
How to integrate Cyan to GIMP 2.10.20 on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by NicholasCureton** [rodlie/cyan#102] [Q&A]
Hi , I was trying to integrate Cyan to GIMP but somehow ... it didn't work.
I don't see Cyan on GIMP menu bar.
I copied Cyan to /usr/bin/ but still didn't work.

Can you please guide me?

**Comment by noriegafam** [comment]
Same here in Fedora Linux
Any one outhere?

**Comment by rodlie** [comment]
As far as I know GIMP in Ubuntu 20.04 does not have working python scripting, not sure about Fedora. Can you load other python scripts in GIMP?

**Comment by noriegafam** [comment]
Hello Rodlie, Thank you for responding. I don't understand the ins and outs of gimp and python scripting. However from following post I understand it is possible. [https://stackoverflow.com/questions/32595938/why-does-my-gimp-python-plug-in-not-work-on-my-ubuntu-linux-box](url). I could be wrong though :)

In any case because you have Linux available version of this it is assumed there is a way to run this under Linux. I am correct? 

this is what I get from the terminal:
$ echo $PYTHONPATH
$/usr/lib64
$ echo $LD_LIBRARY_PATH
$/usr/lib64
$ ./Cyan
./Cyan: error while loading shared libraries: libbz2.so.1.0: cannot open shared object file: No such file or directory

This is how /usr/lib64 looks like:
![image](https://user-images.githubusercontent.com/22824433/144493564-ebc57ee4-e29d-41c1-8916-c40e3f08bc8b.png)

Thank you for time!

**Reply by noriegafam** [reply]
I figured it out.

All I did was to issued this in the terminal:
sudo ln -s /usr/lib64/libbz2.so.1.0.8 /usr/lib64/libbz2.so.1.0

Now it both works as stand alone and as a menu option inside gimp.

**Reply by noriegafam** [reply]
this might be usefull

Cyan-1.2.2-Linux]$ ls /usr/lib64/libbz2.so* -la
lrwxrwxrwx. 1 root root    11 Jan 29  2021 /usr/lib64/libbz2.so -> libbz2.so.1
lrwxrwxrwx. 1 root root    15 Jan 29  2021 /usr/lib64/libbz2.so.1 -> libbz2.so.1.0.8
-rwxr-xr-x. 1 root root 76824 Jan 29  2021 /usr/lib64/libbz2.so.1.0.8

Cyan-1.2.2-Linux]$ sudo ln -s /usr/lib64/libbz2.so.1.0.8 /usr/lib64/libbz2.so.1.0
[sudo] password for : 
Cyan-1.2.2-Linux]$ ls /usr/lib64/libbz2.so* -la
lrwxrwxrwx. 1 root root    11 Jan 29  2021 /usr/lib64/libbz2.so -> libbz2.so.1
lrwxrwxrwx. 1 root root    15 Jan 29  2021 /usr/lib64/libbz2.so.1 -> libbz2.so.1.0.8
lrwxrwxrwx. 1 root root    26 Dec  7 11:20 /usr/lib64/libbz2.so.1.0 -> /usr/lib64/libbz2.so.1.0.8
-rwxr-xr-x. 1 root root 76824 Jan 29  2021 /usr/lib64/libbz2.so.1.0.8

-----------------------------------------

In my opinion this can be marked as solved
